pkgs/applications/editors/helix/default.nix

```diff
@@ -14,6 +14,10 @@ rustPlatform.buildRustPackage rec {
 
   cargoHash = "sha256-/LCtfyDAA2JuioBD/CDMv6OOxM0B9A3PpuVP/YY5oF0=";
 
+  patches = [
+    ./7227-fix-UB-diff-gutter.patch
```

Comment:
Is there a reason you're not using `fetchpatch`?

Reply:
I didn't know `fetchpatch` existed, should I change it to use that?